sycl/test/basic_tests/parallel_for_disable_range_roundup_syclstd.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -sycl-std=2017 %s -o %t.out
```

Comment:
Could you please add a check with different version of sycl-std?

Reply:
OK